net-libs/qxmpp/qxmpp-1.0.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+DESCRIPTION="A cross-platform C++ XMPP client library based on the Qt framework"
+HOMEPAGE="https://github.com/qxmpp-project/qxmpp/"
+SRC_URI="https://github.com/${PN}-project/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug doc opus +speex test theora vpx"
+
+RDEPEND="
+	dev-qt/qtcore
+	dev-qt/qtnetwork[ssl]
+	dev-qt/qtxml
```

Comment:
You need to add a subslot requirement to these deps since they're linked in and are themselves subslotted.

Reply:
no, because Qt subslot description explicitly states that they exist only for private header use.